src/search/landmarks/landmark.h

```diff
@@ -31,7 +31,10 @@ class Landmark {
     bool is_true_in_goal;
     bool is_derived;
 
-    // Cost of achieving the landmark (as determined by the landmark factory)
+    /*
+      Cost of achieving the landmark (as determined by the landmark factory),
+      can be infinite (currently only if *first_achievers* is empty).
```

Comment:
Can you explain what "infinite" means? Integers don't have a notion of infinite as such; we have different conventions in different parts of the code which values to use as a substitute (-1 vs. the maximum int value).